packages/config-plugins/src/android/Version.ts

```diff
@@ -34,6 +34,7 @@ export function setVersionName(config: Pick<ExpoConfig, 'version'>, buildGradle:
 }
 
 export function getVersionCode(config: Pick<ExpoConfig, 'android'>) {
+  // TODO-JJ import this from  @expo/config once new version is published
   return config.android?.versionCode ?? 1;
```

Comment:
config -> config-plugins -> config is a cyclic dep

Reply:
good point. Is there a reason why @expo/config-plugins is being treated as a parent? Seems a bit awkward. 🤔

Reply:
After discussing in slack with @EvanBacon, have come to understand that config-plugins access the expoConfig via `getConfig`. Thus if we set the default values there (which is part of this PR below), we can delete these "default ensuring" functions once this PR is published..